docker/build.sh

```diff
@@ -23,13 +23,16 @@ IIDFILE="${IIDFILE:-$TMP_IIDFILE}"
 PUBLISH="${PUBLISH:-false}"
 BUILDER="${BUILDER:-container}"
 
+export BUILDX_NO_DEFAULT_ATTESTATIONS=""
 # If PUBLISH=false, images only stores in local cache, otherwise they are pushed to th container registry
-BUILDX_NO_DEFAULT_ATTESTATIONS="" docker buildx build --builder="${BUILDER}" --iidfile "${IIDFILE}" --build-arg="PRESTO_VERSION=${VERSION}" \
+docker buildx build --builder="${BUILDER}" --iidfile "${IIDFILE}" \
+    --build-arg="PRESTO_VERSION=${VERSION}" \
+    --build-arg="JMX_PROMETHEUS_JAVAAGENT_VERSION=0.20.0" \
```

Comment:
do you want to make this as a var too?
```
JMX_PROMETHEUS_JAVAAGENT_VERSION="${JMX_PROMETHEUS_JAVAAGENT_VERSION:-0.20.0}"
```
adding the code above after line 24 (maybe a shorter name)
and use the var here

Reply:
the reason for having a var is also for future usage. no need to update the `build.sh` file if a newer version is out. otherwise, you need a PR to update the version.